### DISCUSSION TITLE
Using basicvsr_pp.py with another training setting

### DISCUSSION THREAD

**Discussion by NasrinR791** [open-mmlab/mmediting#1275] [Models]
Hi @ckkelvinchan 

I can train basicvsr++ with default settings using mmediting codes without any problem and it fits in the gpu memory. 
However, when I use  https://github.com/xinntao/EDVR/tree/old_version to train basicvsr_pp.py as the model (replacing BasicVSR++ with the EDVR model), the training stops with cuda memory error even with batch size of 1.

In https://github.com/xinntao/EDVR/tree/old_version in codes/models/networks.py I specify:

from mmedit.models.backbones.sr_backbones.basicvsr_pp import BasicVSRPlusPlus
if which_model == 'BasicVSR_pp':
        netG = BasicVSRPlusPlus(mid_channels=64,
            num_blocks= 7,
            max_residue_magnitude= 10,
            is_low_res_input= False,
            spynet_pretrained = opt_net["spynet_pretrained"],
            cpu_cache_length= 100)

Do you have any idea what is the difference between training with https://github.com/xinntao/EDVR/tree/old_version and mmediting software?

Thank you for your help in advance

**Comment by zengyh1900** [comment]
Please check this issue. @Z-Fran

**Comment by Z-Fran** [comment]
Sorry for the late reply. Can your supply logs of training?
